### ISSUE TITLE
Stories: adopt V1 and V3 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#17067]
## Description
To be in compliance with the Design System and to help make sure our products receive design and accessibility updates, Stories templates needs to be updated to use the most recent versions of all available components.

Example pages:

- [Minneapolis Stories List](https://www.va.gov/minneapolis-health-care/stories/)
  - [Story Detail page](https://www.va.gov/minneapolis-health-care/stories/its-2024-time-to-make-your-new-years-intentions/)
- [Boston Stories List](https://www.va.gov/boston-health-care/stories/)
  - [Story Detail page](https://www.va.gov/boston-health-care/stories/comcasts-veteran-network-and-va-boston-unveil-haven-for-veteran-families/)

### Elements that are not using components that should be 

**Stories List template & Story Detail template**
1. Link
   - These are currently not using a component at all but they should be using [the v1 component](https://design.va.gov/components/link/#download) 
   - Title links on listing template, social links on Detail pages    
   - "View all stories" should also probably use the [active link component](https://design.va.gov/components/link/#active)

~Pagination~
   - ~**NOTE: This is for Stories List templates**~ 
   - ~These are currently not using a component at all but they should be using [the v3 component](https://design.va.gov/components/pagination#examples---v3)~
  

## Acceptance Criteria
- [x] Link component has been added 
- [x] Update and run existing test(s).
- [x] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

## Team
Please check the team(s) that will do this work.

- [ ] CMS Team
- [ ] Public Websites
- [x] Facilities
- [ ] Accelerated Publishing

**Comment by jilladams** [comment]
NOTE: Randi's pagination PR will cover the pagination piece for both Stories and News Releases: https://github.com/department-of-veterans-affairs/content-build/pull/1941.  Max has already reviewed that PR, so we're engaged there now.

**Comment by laflannery** [comment]
Striking out the paging items in the description above based on the above comment, just to make sure we know these don't need to get done here

**Comment by eselkin** [comment]
I estimate a 2

**Comment by randimays** [comment]
This is temporarily blocked until the DST releases a change for this issue: https://dsva.slack.com/archives/C01DBGX4P45/p1714490735773819?thread_ts=1714485532.567929&cid=C01DBGX4P45. The PR to fix it has been merged in as of now, but we're waiting on the DST release.

**Comment by randimays** [comment]
Successfully validated in production. Closing!
